### PR TITLE
12.0 mig edi purchase base

### PR DESCRIPTION
**Pull request by chetanrdhaduk**
Task URL:
https://www.druidoo.io/web#id=1742&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

**Comment by ivantodorovich** [comment]
Can you check why tests are failing with this module?

File: edi_purchase_base/models/purchase.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class PurchaseOrder(models.Model):
     date_planned = fields.Datetime(
         string="Scheduled Date",
         compute="_compute_date_planned",
-        required=True,
+        required=False,
         index=True,
         oldname="minimum_planned_date",
```
Comment (ivantodorovich) [inline edi_purchase_base/models/purchase.py]:
```suggestion
```
This is not needed. If you find more occurrences like this, remove them

File: edi_purchase_base/models/purchase.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class PurchaseOrder(models.Model):
     date_planned = fields.Datetime(
         string="Scheduled Date",
         compute="_compute_date_planned",
-        required=True,
+        required=False,
```
Comment (ivantodorovich) [inline edi_purchase_base/models/purchase.py]:
```suggestion
```
This is the default, right?

File: edi_purchase_base/models/picking_update.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# @author: Druidoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+from odoo import models, fields, api
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class PickingEdi(models.Model):
+    _name = "picking.edi"
```
Comment (ivantodorovich) [inline edi_purchase_base/models/picking_update.py:11]:
```suggestion
    _name = "picking.edi"
    _description = "Picking EDI"
```

File: edi_purchase_base/models/supplier_price_list.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2016-Today: Druidoo (<http://www.druidoo.net/>)
+# @author: Druidoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+from odoo import models, fields
+import odoo.addons.decimal_precision as dp
+
+
+class SupplierPriceList(models.Model):
+    _name = "supplier.price.list"
```
Comment (ivantodorovich) [inline edi_purchase_base/models/supplier_price_list.py:10]:
```suggestion
    _name = "supplier.price.list"
    _description = "Supplier Price List"
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: edi_purchase_base/models/purchase.py (hunk 2 above)
Comment (chetanrdhaduk) [reply edi_purchase_base/models/purchase.py]: Yes, It's default

**Comment by ivantodorovich** [comment]
The model picking.edi has no access rules, consider adding one. E.g. access_picking_edi,access_picking_edi,model_picking_edi,base.group_user,1,0,0,0

**Comment by ivantodorovich** [comment]
If you can fix those warnings it would be great, that way we can track real warnings because apparently sometimes they report very critical things

```
2020-03-05 14:01:56,817 6424 WARNING openerp_test odoo.models: The model edi.mapping.lines has no _description
2020-03-05 14:01:56,817 6424 WARNING openerp_test odoo.models: The model edi.price.mapping has no _description
2020-03-05 14:01:56,818 6424 WARNING openerp_test odoo.models: The model edi.ble.mapping has no _description
2020-03-05 14:01:56,818 6424 WARNING openerp_test odoo.models: The model edi.config.system has no _description
```

**Pull request merged**